apps/wallet-dashboard/components/VirtualList.tsx

```diff
@@ -34,8 +34,7 @@ export function VirtualList<T>({
 }: VirtualListProps<T>): JSX.Element {
     const containerRef = useRef<HTMLDivElement | null>(null);
     const virtualizer = useVirtualizer({
-        // Render one more item if there is still pages to be fetched
-        count: hasNextPage ? items.length + 1 : items.length,
+        count: items.length,
```

Comment:
we should keep the logic we had before but explain why eg `Render one more item to fit the "loading" message if there are more pages to be fetched`